test/unit_test/agent/tools/test_querit_unit.py

```
@@ -0,0 +1,560 @@
+        {
+            "title": "123",
+            "url": "456",
+            "content": "789",
+            "score": 1,
+        }
+    ]
+    assert outputs["json"] == raw_response
+
+
+def test_api_key_is_redacted_from_errors_and_logs(monkeypatch, caplog):
+    secret = "secret-test-key"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "secret"

(S105)